### ISSUE TITLE
Add basic build instructions to the main README.md

### ISSUE THREAD

**Issue by mrmowgli** [argouml-tigris-org/argouml#44]
Until there is a formal release process, consider adding the basic build instructions to the main README.md file.

Even basic instructions would be helpful, and any tips or tricks to get it working. For example,

- [ ] Minimum Java version (Sun/OpenJDK)
- [ ] Maximum Java version
- [ ] Maven repositories needed
- [ ] Static assets needed
- [ ] Maven build / run / test commands
- [ ] Any changes to HOSTS files needed to offset the Tigris domain changes

**Comment by linustolke** [comment]
Yes, there is some more work before we can get the releases in place. Having some quick instructions in the README.md along these lines would make life simpler primarily for the developers. Please go ahead and  push this change with this (to Gerrithub).

**Comment by TimoGoetze** [comment]
We would appreciate this, too. It is hard to get it running without doc as maven doesnt do everything needed.

**Comment by linustolke** [comment]
@stcarrez I have moved your Pull Request as a change in Gerrithub and added some comments there.

**Comment by linustolke** [comment]
The hosts file changes, were they not solved by https://github.com/argouml-tigris-org/argouml/commit/bf30ff1c0570ad41b4290f3491f83a5c7c618032 and https://github.com/argouml-tigris-org/parentpom/commit/c2e240733cd397663598b3a067f3aaebfa47769f ?
